<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_dialog</class>
 <widget class="QDialog" name="main_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>WinBoot</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../Pictures/assets/favicon.png</normaloff>../../../Pictures/assets/favicon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #121212;</string>
  </property>
  <widget class="QLabel" name="drive_label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>220</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>111</width>
     <height>111</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>111</width>
     <height>111</height>
    </size>
   </property>
   <property name="pixmap">
    <pixmap>../assets/usb_drive.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QPushButton" name="start_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>735</x>
     <y>360</y>
     <width>151</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>151</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>151</width>
     <height>50</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: #FFFFFF;
	font-size: 20px;
	border-radius: 5px;
	background-color: rgb(0, 120, 212);
}

QPushButton:hover{
	background-color: rgb(1, 93, 163);
}

QPushButton:pressed{
	background-color: rgb(0, 144, 255);
}

QPushButton:disabled{
	background-color: rgba(0, 120, 212, 0.65);
}</string>
   </property>
   <property name="text">
    <string>Start!</string>
   </property>
  </widget>
  <widget class="QLabel" name="instruction_label_third">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>590</y>
     <width>1024</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1024</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1024</width>
     <height>30</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color: #FFFFFF;
	font-size: 18px;
	background: none;
	selection-color: rgb(255, 255, 255);
	selection-background-color: rgb(0, 120, 212);
}</string>
   </property>
   <property name="text">
    <string>3. Please select Windows 10/11 .iso file, otherwise it may crash your drive.</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="indent">
    <number>210</number>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="add_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>111</width>
     <height>111</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>111</width>
     <height>111</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>111</width>
     <height>111</height>
    </size>
   </property>
   <property name="pixmap">
    <pixmap>../assets/add_file.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QPushButton" name="drive_button">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>360</y>
     <width>151</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>151</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>151</width>
     <height>50</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: #FFFFFF;
	font-size: 20px;
	border-radius: 5px;
	background-color: rgb(0, 120, 212);
}

QPushButton:hover{
	background-color: rgb(1, 93, 163);
}

QPushButton:pressed{
	background-color: rgb(0, 144, 255);
}

QPushButton:disabled{
	background-color: rgba(0, 120, 212, 0.65);
}</string>
   </property>
   <property name="text">
    <string>Select Drive</string>
   </property>
  </widget>
  <widget class="QLabel" name="flash_label">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>212</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>121</width>
     <height>121</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>121</width>
     <height>121</height>
    </size>
   </property>
   <property name="pixmap">
    <pixmap>../assets/main_flash.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QPushButton" name="add_button">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>360</y>
     <width>151</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>151</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>151</width>
     <height>50</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: #FFFFFF;
	font-size: 20px;
	border-radius: 5px;
	background-color: rgb(0, 120, 212);
}

QPushButton:hover{
	background-color: rgb(1, 93, 163);
}

QPushButton:pressed{
	background-color: rgb(0, 144, 255);
}

QPushButton:disabled{
	background-color: rgba(0, 120, 212, 0.65);
}</string>
   </property>
   <property name="text">
    <string>Add File</string>
   </property>
  </widget>
  <widget class="QLabel" name="instruction_label_second">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>550</y>
     <width>1024</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1024</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1024</width>
     <height>30</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color: #FFFFFF;
	font-size: 18px;
	background: none;
	selection-color: rgb(255, 255, 255);
	selection-background-color: rgb(0, 120, 212);
}</string>
   </property>
   <property name="text">
    <string>2. This process will delete all the data of your drive.</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="indent">
    <number>210</number>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="logo_label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>272</x>
     <y>0</y>
     <width>480</width>
     <height>180</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>480</width>
     <height>180</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>480</width>
     <height>180</height>
    </size>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="pixmap">
    <pixmap>../assets/logo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::NoTextInteraction</set>
   </property>
  </widget>
  <widget class="QLabel" name="instruction_label_first">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>510</y>
     <width>1024</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1024</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1024</width>
     <height>30</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color: #FFFFFF;
	font-size: 18px;
	background: none;
	selection-color: rgb(255, 255, 255);
	selection-background-color: rgb(0, 120, 212);
}</string>
   </property>
   <property name="text">
    <string>1. Take data backup of the USB drive.</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <property name="indent">
    <number>210</number>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="instruction_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>465</y>
     <width>1024</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1024</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1024</width>
     <height>35</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color: #FFFFFF;
	font-size: 20px;
	background: none;
	selection-color: rgb(255, 255, 255);
	selection-background-color: rgb(0, 120, 212);
}</string>
   </property>
   <property name="text">
    <string>---------------------------------------- Instructions ----------------------------------------</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QFrame" name="drive_frame">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>112</y>
     <width>600</width>
     <height>425</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>425</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>425</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #000000;
border-radius: 10px;
background-color: #FFFFFF;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLabel" name="select_drive_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>400</width>
      <height>45</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>45</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>45</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font: 18pt &quot;Ubuntu&quot;;
	font-size: 30px;
	color: rgb(0, 0, 0);
	selection-color: rgb(255, 255, 255);
	selection-background-color: rgb(0, 120, 212);
}</string>
    </property>
    <property name="text">
     <string>Select Drive</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="indent">
     <number>10</number>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QPushButton" name="refresh_button">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>28</y>
      <width>36</width>
      <height>36</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:pressed{
	border-width: 3px;
	border-style: solid;
	border-color: rgb(136, 138, 133);
	border-radius: 15px;
}

QPushButton:hover{
	background-color: rgb(243, 243, 243);
}</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../assets/refresh.png</normaloff>../assets/refresh.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>36</width>
      <height>36</height>
     </size>
    </property>
    <property name="shortcut">
     <string>F5</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="drive_table">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>540</width>
      <height>270</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>540</width>
      <height>270</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>540</width>
      <height>270</height>
     </size>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget{
	color: #000000;
	background-color: rgb(238, 238, 236);
	alternate-background-color: rgb(255, 255, 255);
}

QHeaderView::section{
	border: none;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="tabKeyNavigation">
     <bool>true</bool>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::NoDragDrop</enum>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectItems</enum>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>false</bool>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>100</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>40</number>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Size</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Location</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="apply_button">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>362</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>50</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: #FFFFFF;
	font-size: 20px;
	border-radius: 5px;
	background-color: rgb(0, 120, 212);
}

QPushButton:hover{
	background-color: rgb(1, 93, 163);
}

QPushButton:pressed{
	background-color: rgb(0, 144, 255);
}

QPushButton:disabled{
	background-color: rgba(0, 120, 212, 0.65);
}</string>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel_button">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>362</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>50</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: #FFFFFF;
	font-size: 20px;
	border-radius: 5px;
	background-color: rgb(209, 26, 42);
}

QPushButton:hover{
	background-color: rgb(170, 22, 35);
}

QPushButton:pressed{
	background-color: rgb(218, 21, 38);
}

QPushButton:disabled{
	background-color: rgba(209, 16, 42, 0.65);
}</string>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="file_name_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>360</y>
     <width>251</width>
     <height>90</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>251</width>
     <height>90</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>251</width>
     <height>90</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color: #FFFFFF;
}</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="drive_name_label">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>360</y>
     <width>251</width>
     <height>90</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>251</width>
     <height>90</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>251</width>
     <height>90</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	color: #FFFFFF;
}</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <zorder>drive_name_label</zorder>
  <zorder>file_name_label</zorder>
  <zorder>drive_label</zorder>
  <zorder>start_button</zorder>
  <zorder>instruction_label_third</zorder>
  <zorder>add_label</zorder>
  <zorder>drive_button</zorder>
  <zorder>flash_label</zorder>
  <zorder>add_button</zorder>
  <zorder>instruction_label_second</zorder>
  <zorder>logo_label</zorder>
  <zorder>instruction_label_first</zorder>
  <zorder>instruction_label</zorder>
  <zorder>drive_frame</zorder>
 </widget>
 <tabstops>
  <tabstop>add_button</tabstop>
  <tabstop>drive_button</tabstop>
  <tabstop>start_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
